### ISSUE TITLE
Nested arrays trashing on update

### ISSUE THREAD

**Issue by davidmerrique** [WebReflection/hyperHTML#86]
I have an array of items that I'm rendering. 

I want each of those items to have the same `<select>`'s to choose from, but when I call `update()` it trashes all of those selects.

Here's an example using a fork of one of your CodePens:

https://codepen.io/davidmerrique/pen/MvOvwY?editors=0010

If you change any of the selects and click update, all of the selects are trashed.

Is there a way that I can prevent this?

Thank you

**Comment by WebReflection** [comment]
yes, you are using an array which means HTML, expected behavior.

What you need instead is either wire every option node to that item in a unique way
https://codepen.io/WebReflection/pen/JyOymp?editors=0010

or to create a selected chunk of text in that string.

TL;DR if an interpolation is an array of strings, that's just injected as HTML, it has no special meanings.

To have special meanings you need wires. Usually you wire the select, not it's options, but in this case if you want to keep current nodes you have options.

The one I've used in the pen is the easy one, but it's not too elegant.
The one setting options as selected or not might trash the options anyway ... use wires and :IDs, they are there to solve these cases.

Try to avoid passing HTML as Array as much as you can if you don't want to trash layout and you want a safer content.
